ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -31,32 +31,49 @@ module.exports = Vue.extend({
     'insert-code': insertCodeComponent,
     'draggable-statement-completion': draggableStatementCompletionComponent
   },
+  watch : {
+    interactiveIdOrSlug: async function() {
```

Comment:
Same comment as above - the error handling code could be consolidated to the getINteractiveData call.